coalib/parsing/DefaultArgParser.py

```diff
@@ -36,8 +36,7 @@ def default_arg_parser(formatter_class=None):
 code, regardless of the programming languages you use.
 
 To find out what kind of analysis coala offers for the languages you use, visit
-<https://github.com/coala/bear-docs/blob/master/README.rst#supported-languages>
-or run:
+http://coala.io/languages, or run::
```

Comment:
The <> is not necessary and looks weird when viewed in the docs
<img width="636" alt="screen shot 2017-01-01 at 11 00 25 pm" src="https://cloud.githubusercontent.com/assets/5844587/21585489/3e4a314e-d076-11e6-9cf3-10054250c508.png">